ies-route53/recipes/add.rb

```diff
@@ -0,0 +1,27 @@
+require 'net/http'
+Chef::Resource.send(:include, EasyBib)
+
+include_recipe 'route53'
+
+host_name = get_hostname(node, true)
+stack_name = get_normalized_cluster_name(node)
+zone_name = node['ies-route53']['zone']['name']
+region_id = Net::HTTP.get(URI.parse('http://169.254.169.254/latest/meta-data/placement/availability-zone'))
```

Comment:
Maybe not quite yet... There is still a tailing `a` for the availability zone within the region. This letter needs to be trimmed.